pkg/util/azureclient/mgmt/redhatopenshift/openshiftclusters.go

```diff
@@ -41,7 +41,7 @@ func NewOpenShiftClustersClient(subscriptionID string, authorizer autorest.Autho
 			},
 		}
 	} else {
-		client = redhatopenshift.NewOpenShiftClustersClient(subscriptionID)
+		client = redhatopenshift.NewOpenShiftClustersClientWithBaseURI("https://management.azure.com", subscriptionID)
```

Comment:
why not save changes like this for a different PR?

Reply:
I've copied this commit to a new branch and am dropping it from this one.